style.css

```diff
@@ -0,0 +1,284 @@
+html {
+	font-size: 10px;
+	scroll-behavior: smooth;
+	font-family: 'Varela Round', sans-serif;
+	box-sizing: border-box;
+}
+
+nav {
+	position: relative;
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	gap: 30px;
+	background-color: #BD4B4B;
+}
+
+nav div a {
+	padding: 30px;
+}
+
+nav::after {
+ 	position: absolute;
+	bottom: -50px;
+ 	left: 0px;
+ 	width: 100%;
+	content:" ";
+ 	background:
+ 	radial-gradient(circle at 50% 0%, #BD4B4B 25%, #EEEEEE 26%, #EEEEEE 40%);
+ 	background-color: #EEEEEE;
+ 	background-size: 30px 100px;
+ 	height: 50px;
+}
+nav img {
+	width: 175px;
+}
+
+.navigation-links {
+	color: #ffffff;
+	text-decoration: none;
+	font-size: 3rem;
+}
+
+header {
+	background-color: #EEEEEE;
+	height: 90vh;
+	color: #000000;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	padding: 50px 0;
+}
+
+header p {
+	font-size: 6rem;
+	display: flex;
+	position: relative;
+}
+
+.sparkle-item {
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+	height: 100px;
+	width: 500px;
+	margin: 0;
+}
```

Comment:
all of these apply to the exact same element - should just choose one and put all the rules in there